_includes/landing-navbar-hipaa.html

```diff
@@ -63,7 +63,7 @@
               <ul class="dropdown-menu dropdown-menu-left" role="menu">
                 <li>
                   <a
-                    href="{{ landing_page_base_url }}/technical-details#reference-infra"
+                    href="{{ landing_page_base_url }}/technical-details/#reference-infra"
```

Comment:
Thanks for the quick fix Zack! Note there's one more piece to this fix: updating the text from "Reference Infrastructure" to "Reference Architecture" a few lines below.

Reply:
Thanks, Jim :+1: Note that the use of "Infrastructure" there was actually intentional, since we were not sure that newcomers to the page would understand what an "Architecture" is in the context of Gruntwork, whereas we figured "Infrastructure" was generic enough to be generally understood at first glance, hopefully. 

That said, I'm happy to switch it back, too.

Reply:
Updated in dc174ee

Reply:
I don't feel too strongly about it, but the page itself has the title "Reference Architecture," and that's what we use elsewhere, so perhaps consistency is the better route?